bitnami/mariadb/Chart.yaml

```diff
@@ -26,4 +26,4 @@ sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/mariadb
   - https://github.com/prometheus/mysqld_exporter
   - https://mariadb.org
-version: 11.3.5
+version: 11.3.6
```

Comment:
As this is a new feature, could you please bump a new minor version `11.4.0`?

Reply:
Sure, it's done.